src/SFA.DAS.DATA.MANAGEMENT/Tables/ASDataPL_GA_SessionData.sql

```diff
@@ -0,0 +1,35 @@
+﻿CREATE TABLE [ASData_PL].[GA_SessionData](
+	[GASD_Id]						[bigint] IDENTITY(1,1) NOT NULL,
+	[FullVisitorId]					[nvarchar](500) NULL,
+	[ClientId]						[nvarchar](500) NULL,
+	[VisitId]						[nvarchar](500) NULL,
+	[VisitNumber]					[nvarchar](500) NULL,
+	[VisitStartDateTime]			[nvarchar](500) NULL,
+	[VisitDate]						[nvarchar](500) NULL,
+	[VisitorId]						[nvarchar](500) NULL,
+	[UserId]						[nvarchar](500) NULL,
+	[Hits_Page_PagePath]			[nvarchar](max) NULL,	
+	[Hits_Time]						[nvarchar](500) NULL,
+	[Hits_IsEntrance]				[nvarchar](500) NULL,
+	[Hits_IsExit]					[nvarchar](500) NULL,
+	[EmployerId]					[nvarchar](500) NULL,
+	[ID2]							[nvarchar](500) NULL,
+	[ID3]							[nvarchar](500) NULL,
+	[ESFAToken]						[nvarchar](500) NULL,
+	[EventCategory]					[nvarchar](max) NULL,
+	[EventAction]					[nvarchar](max) NULL,	
+	[EventLabel_ESFAToken]			[nvarchar](500) NULL,
+	[EventLabel_Keyword]			[nvarchar](2000) NULL,
+	[EventLabel_Postcode]			[nvarchar](500) NULL,
+	[EventLabel_WithinDistance]		[nvarchar](500) NULL,
+	[EventLabel_Level]				[nvarchar](500) NULL,	
+	[CD_ClientId]					[nvarchar](500) NULL,
+	[CD_SearchTerms]				[nvarchar](2000) NULL,
+	[CD_UserId]						[nvarchar](500) NULL,
+	[CD_LevyFlag]					[nvarchar](500) NULL,
+	[CD_EmployerId]					[nvarchar](500) NULL,
+	[CD_ESFAToken]					[nvarchar](500) NULL,
+	[CD_LegalEntityId]				[nvarchar](500) NULL,	
+	[GA_ImportDate]					[datetime2](7) DEFAULT getdate()
+) 
```

Comment:
Do we want to put file name in PL for Traceability, for any reverse engineering stuff?

Reply:
In the new implementation we are truncating the data from staging and on top not archiving files. Its not very helpful updating the data filename to PL time.  As a std practice keep updating the import datetime in PL time. If not happy to add import data filename

Reply:
Agree, If we are not archiving then it's not of great use having it.

As long as there is something we can track back in case of issues. If GA_ImportDate helps with it, Then we are good.